src/named_tuple.cr

```diff
@@ -26,7 +26,7 @@ struct NamedTuple
   # Creates a named tuple that will contain the given arguments.
   #
   # This method is useful in macros and generic code because with it you can
-  # creates empty named tuples, something that you can't do with a tuple literal.
+  # create empty named tuples, something that you can't do with a tuple literal.
```

Comment:
Maybe a good opportunity to rephrase slight? :)

> With a named tuple literal you cannot create an empty named tuple. This method doesn't have this limitation, which makes it especially useful in macros and generic code.

Reply:
I also thought about rephrasing, but couldn't come up with a concise solution. Yours is not better than the original one imho :grin:

Reply:
Sorry, I should have mentioned, that just rephrasing doesn't add much value for me. I would have preferred adding more context

Reply:
> *options* can be empty, so this method can create an empty `NamedTuple`. A tuple literal can't express this.

Reply:
I took @jhass rephrasing and changed the order of the examples to emphasize the method's use-case. Have look